### PR TITLE
media-plugins/kodi-*: add ebuilds for all add-ons currently available in kodi upstream

### PR DESCRIPTION
**Pull request by candrews**
https://bugs.gentoo.org/show_bug.cgi?id=555792

This PR includes ebuilds for all add-ons included in Kodi. All add-ons have live ebuilds; add-ons that have release versions have versioned ebuilds too. I'm contacting upstreams asking them to make releases if they haven't already done so or are behind (I don't think that should block this PR; version bumps can be handled separately later).

Feedback very welcome!

This work is based on that done by @herrnst at https://github.com/herrnst/gentoo-kodi-overlay

repoman currently complains that these ebuilds can't be keyworded ~x86; that issue will be fixed when https://github.com/gentoo/gentoo/pull/3194 is merged.

### CODE REVIEW

File: media-plugins/kodi-pvr-vdr-vnsi/kodi-pvr-vdr-vnsi-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/kodi-pvr/pvr.vdr.vnsi.git"
```
Comment (cjmayo) [inline media-plugins/kodi-pvr-vdr-vnsi/kodi-pvr-vdr-vnsi-9999.ebuild]: Location just got moved to https://github.com/FernetMenta/pvr.vdr.vnsi/:
https://github.com/kodi-pvr/pvr.vdr.vnsi/commit/36023a3675f4a52c43676951046540f1d274061d

Already rolling on to Kodi 18:
https://github.com/FernetMenta/pvr.vdr.vnsi/commit/4d17aaf4ff4d94f9d355365dc516a6859f685037

2.6.9 has been working fine for me on Kodi 17 beta 6 (and seems OK on beta 7)
https://github.com/FernetMenta/pvr.vdr.vnsi/commit/4b06d83e22ac7a30629dd0edf81c5b985cb0133f

File: media-plugins/kodi-inputstream-adaptive/Manifest [hunk 2]
```diff
@@ -0,0 +1 @@
+EBUILD kodi-inputstream-adaptive-9999.ebuild 492 SHA256 604c88d1cd147222a90e6f77320f9ea8481fdd4e38c2c7f86bc0a0d12aa7a837 SHA512 de66e63289a25d7ba5573197387d2550ce81f0904d88f0c11ca9bc956abba36711f81eb1595fdb53538a7365f5aa82d1b39de14ea569d5eac720b8413ec02c2d WHIRLPOOL 4718a45e77d312558b8582dc1bdf10d3382ea3a43d109772841cf1f2aff49305e9758e2c711e97329f5c3f8bb443f9d86b87c53520721490097cd707242be5a8
```
Comment (SoapGentoo) [inline media-plugins/kodi-inputstream-adaptive/Manifest]: @candrews these don't belong in the git tree. Could you see to it that all the `EBUILD` lines disappear from the Manifests?

Comment (candrews) [inline media-plugins/kodi-inputstream-adaptive/Manifest]: Sure, should I do a PR for that?

Also, why does `ebuild manifest` add these lines if they shouldn't be there?

Comment (SoapGentoo) [inline media-plugins/kodi-inputstream-adaptive/Manifest]: @candrews beats me

Comment (floppym) [inline media-plugins/kodi-inputstream-adaptive/Manifest]: @candrews Those lines should not be added by `ebuild manifest` if thin manifests are enabled for the repository. Have you altered metadata/layout.conf to remove `thin-manifests = true`?

Comment (SoapGentoo) [inline media-plugins/kodi-inputstream-adaptive/Manifest]: @candrews also, no need to fix it anymore, I fixed it already

Comment (candrews) [inline media-plugins/kodi-inputstream-adaptive/Manifest]: I just have a git checkout of the Gentoo/Gentoo repo. I run `ebuild kodi-inputstream-adaptive-9999.ebuild` from the directory that contains it. I haven't touched metadata/layout.conf or anything else unless expressed in this PR, so I don't understand how this undesirable manifest entry comes to be.

**Pull request closed without merging**